src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+#: src/view/com/modals/VerifyEmail.tsx:255
+#: src/view/com/modals/VerifyEmail.tsx:261
+#: src/view/com/util/post-ctrls/RepostButton.tsx:166
+#: src/view/screens/Search/Search.tsx:910
+msgid "Cancel"
+msgstr "Annuleer"
+
+#: src/view/com/modals/CreateOrEditList.tsx:340
+#: src/view/com/modals/DeleteAccount.tsx:174
+#: src/view/com/modals/DeleteAccount.tsx:296
+msgctxt "action"
+msgid "Cancel"
+msgstr "Annuleer"
+
+#: src/view/com/modals/DeleteAccount.tsx:170
+#: src/view/com/modals/DeleteAccount.tsx:292
+msgid "Cancel account deletion"
+msgstr "Annuleeer accountverwijdering"
+
+#: src/view/com/modals/ChangeHandle.tsx:137
+msgid "Cancel change handle"
+msgstr "Annuleer wijzigen handle"
+
+#: src/view/com/modals/CropImage.web.tsx:94
+msgid "Cancel image crop"
+msgstr "Annuleer afbeelding bijsnijden"
+
+#: src/view/com/util/post-ctrls/RepostButton.tsx:161
+msgid "Cancel quote post"
+msgstr "Annuleer citaatbericht"
+
+#: src/screens/Deactivated.tsx:155
+msgid "Cancel reactivation and log out"
+msgstr "Annuleer heractivering en meld af"
+
+#: src/view/com/modals/ListAddRemoveUsers.tsx:88
+#: src/view/screens/Search/Search.tsx:902
+msgid "Cancel search"
+msgstr "Annuleer zoeken"
+
+#: src/view/com/modals/LinkWarning.tsx:106
+msgid "Cancels opening the linked website"
+msgstr "Annuleert openen van de gelinkte website"
+
+#: src/state/shell/composer/index.tsx:82
+#: src/view/com/util/post-ctrls/PostCtrls.tsx:113
+#: src/view/com/util/post-ctrls/PostCtrls.tsx:154
+#: src/view/com/util/post-ctrls/PostCtrls.tsx:190
+msgid "Cannot interact with a blocked user"
+msgstr "Kan niet communiceren met een geblokkeerde gebruiker"
+
+#: src/view/com/composer/videos/SubtitleDialog.tsx:56
+msgid "Captions & alt text"
+msgstr "Ondertiteling & alt-tekst"
+
+#: src/view/com/composer/videos/SubtitleDialog.tsx:133
+msgid "Captions (.vtt)"
+msgstr "Ondertiteling (.vtt)"
+
+#: src/view/com/modals/VerifyEmail.tsx:160
+msgid "Change"
+msgstr "Wijzigen"
+
+#: src/view/screens/Settings/index.tsx:342
+msgctxt "action"
+msgid "Change"
+msgstr "Wijzigen"
+
+#: src/view/com/modals/ChangeHandle.tsx:149
+#: src/view/screens/Settings/index.tsx:696
+msgid "Change Handle"
+msgstr "Wijzig handle"
+
+#: src/view/com/modals/ChangePassword.tsx:142
+#: src/view/screens/Settings/index.tsx:741
+msgid "Change Password"
+msgstr "Wijzig wachtwoord"
+
+#: src/view/com/modals/ChangeEmail.tsx:104
+msgid "Change Your Email"
+msgstr "Wijzig uw e-mailadres"
+
+#: src/components/dialogs/VerifyEmailDialog.tsx:147
+msgid "Change email address"
+msgstr "Wijzig e-mailadres"
+
+#: src/view/screens/Settings/index.tsx:685
+msgid "Change handle"
+msgstr "Wijzig handle"
+
+#: src/view/com/modals/VerifyEmail.tsx:155
+msgid "Change my email"
+msgstr "Wijzig mijn e-mailadres"
+
+#: src/view/screens/Settings/index.tsx:730
+msgid "Change password"
+msgstr "Wijzig wachtwoord"
+
+#: src/view/com/composer/select-language/SuggestedLanguage.tsx:73
+msgid "Change post language to {0}"
+msgstr "Wijzig berichttaal naar {0}"
```

Comment:
```suggestion
msgstr "Wijzig taal van het bericht naar {0}"
```

Probably more clear

Reply:
Left unchanged given preference for combining words in Dutch.